src/asciidoctorFindIncludeProcessor.ts

```diff
@@ -0,0 +1,49 @@
+interface IncludeEntry {
+  name: string
+  position: number
+  length: string
+}
+
+interface IncludeItem {
+  [index: number]: IncludeEntry
+}
+
+interface IncludeItems extends Array<IncludeItem>{}
+
+let baseDocIncludes: IncludeItems[] = []
```

Comment:
I think it would be better to use an array of `IncludeEntry` rather than an array of `Array<IncludeItem>`

```js
[
  { name: 'a.adoc', position: 5, length: 6 },
  { name: 'github.adoc', position: 10, length: 11 },
  { name: 'windows.adoc', position: 16, length: 12 },
  { name: 'b.adoc', position: 27, length: 6 },
  { name: 'a.adoc', position: 31, length: 6 },
  { name: 'c.adoc', position: 45, length: 6 }
]
```

Reply:
Yes, it is better. This was my hacky way of getting an index position easily. I will improve.